envi/common.py

```diff
@@ -1,18 +1,25 @@
 import logging
 
 EMULOG = 9
-LOG_LEVELS = ('EMULOG', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL')
+LOG_LEVELS = (
+        'CRITICAL',
+        'ERROR',
+        'WARNING',
+        'INFO',
+        'DEBUG',
+        'EMULOG',
+        )
 LOG_FORMAT = '%(asctime)s:%(levelname)s:%(name)s: %(message)s' \
              '[%(filename)s:%(funcName)s:%(lineno)s:%(threadName)s]'
 
+logging.basicConfig(level=logging.WARNING, format=LOG_FORMAT)
```

Comment:
as a consumer of the vivisect library, its important to me that the library not configure the logging subsystem. using `logging.basicConfig` stomps on any configuration provided by an application, such as the verbosity and format.

can we move this line into `vdbbin` and `vivbin`?

Reply:
yes!

Reply:
please review again.  i think this should solve the concerns mentioned by each of us.